e2e/test/scenarios/onboarding/setup/setup.cy.spec.ts

```diff
@@ -239,11 +241,20 @@ describe("scenarios > setup", () => {
         .clear()
         .type(strongPassword, { delay: 0 })
         .blur();
-
-      cy.findByText("Next").click();
     });
 
-    cy.button("Next").click();
+    cy.findByLabelText(/What should we call you/)
+      .button("Next")
+      .click();
+
+    // make sure this bit of the form loads before clicking next
+    cy.findByLabelText(/What will you use Metabase for/).findByText(
+      /Let us know your plans/,
+    );
+
+    cy.findByLabelText(/What will you use Metabase for/)
+      .button("Next")
+      .click();
 
     cy.findByTestId("database-form").within(() => {
```

Comment:
in CI it was never finding this form, I think it was clicking the same "next" button twice, instead of properly advancing through two stages of setup process